src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/HenleggBehandling/useHenleggBehandling.ts

```diff
@@ -33,6 +33,13 @@ const useHenleggBehandling = (lukkModal: () => void) => {
             }),
             begrunnelse: useFelt({
                 verdi: '',
+                valideringsfunksjon: (felt: FeltState<string>) =>
+                    felt.verdi.length > 5
```

Comment:
5 er litt magisk her, korfor akkurat 5?

Reply:
Godt spørsmål! Ville sette en grense som sikrer at de ikke bare skriver typ "." eller "xyz", men som også tillater ett ord eller to ord hvis dette er beskrivende ord. Så endte egentlig opp med 5 for å sikre at valideringen ikke ble kompleks, men samtidig ga den verdien jeg var på jakt etter 😄

Reply:
Haha, okei.